pkg/devfile/adapters/common/command.go

```diff
@@ -73,7 +81,7 @@ func getCommandFromFlag(data data.DevfileData, groupType common.DevfileCommandGr
 	commands := data.GetCommands()
 
 	for _, command := range commands {
-		if command.Exec.Id == commandName {
+		if command.GetID() == commandName {
```

Comment:
Can we add some comp cmd unit test cases for this func?

Reply:
Done